release-notes/6.0/supported-os.json

```diff
@@ -138,6 +138,39 @@
                         "3.12"
                     ]
                 },
+                {
+                    "id": "centos",
+                    "name": "CentOS",
+                    "link": "https://centos.org/",
+                    "lifecycle": "https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/",
+                    "architectures": [
+                        "x64"
+                    ],
+                    "supported-versions": [
+                    ],
+                    "unsupported-versions": [
+                        "8",
+                        "7"
+                    ],
+                    "notes": [
+                        "The CentOS project has moved to [supporting CentOS Stream as its future](https://blog.centos.org/2020/12/future-is-centos-stream/). Users can consider [migrating to Red Hat Enterprise Linux](https://www.redhat.com/en/blog/centos-linux-has-reached-its-end-life-eol) or another distro."
+                    ]
+                },
+                {
+                    "id": "centos-stream",
+                    "name": "CentOS Stream",
+                    "link": "https://centos.org/",
+                    "architectures": [
+                        "x64"
```

Comment:
Builds of [.NET 8](https://kojihub.stream.centos.org/koji/buildinfo?buildID=61792) in CentOS Stream 9 are available for arm64, ppc64le, s390x and x64.

Reply:
Sure. That would be in the .NET 8 variant of this file, right?

Reply:
The point of removing all the RHEL compatible builds (beyond RHEL) is to reduce noise with a simple statement. If there are RHEL-compatible distros with their own .NET 8 builds, then that's a different story. I'd be happy for CentOS Stream 9 (or any other distro) to be added to the .NET 8+ variants of these files if they provide their own builds.  Same thing for [linux.md](https://github.com/dotnet/core/blob/main/linux.md#packages).

Reply:
> That would be in the .NET 8 variant of this file, right?

Yes. I missed that this was .NET 8 at first. For .NET 6/CentOS Stream 9, arm64, s390x and x64 are supported.

Reply:
I updated all .NET 6, 8, and 9 files for CentOS Steam. PTAL.

Reply:
Was this supported on .NET 7? If yes, then we should update those as well to at least have the history correct.